src/components/accordion/Accordion.stories.jsx

```diff
@@ -117,21 +85,43 @@ export const Controlled = (args: any) => {
     <Accordion onChange={handleChange} expanded={expanded} {...props}>
       {CONTROLLED_ACCORDION_IDS.map(id => (
         <AccordionItem title={copy.title} key={id} id={id}>
-          {copy.description} <CallToAction url={copy.url} cta={copy.cta} />
+          {copy.description}
         </AccordionItem>
       ))}
     </Accordion>
   );
 };
 
-Controlled.args = {
+Expanded.args = {
   expanded: CONTROLLED_ACCORDION_IDS[0],
+  reduceMotion: true,
 };
 
-Controlled.argTypes = {
-  expanded: {
-    control: {type: 'select', options: CONTROLLED_ACCORDION_IDS},
-  },
-  allowMultiple: {control: null},
-  defaultExpanded: {control: null},
+const spacings = [
+  'xxs',
+  'xs',
+  's',
+  'm',
+  'l',
+  'xl',
+  'xxl',
+  'xxxl',
+  'xxxxl',
+  'none',
+];
+
+export const Spacing = (args: any) => (
+  <div>
+    {spacings.map(spacing => (
+      <StoryVariant key={spacing} label={`spacing - ${spacing}`}>
```

Comment:
I don't like this idea of story variant. It bloats code preview and feels redundant. In the context of accordion it's an overkill. We don't need to present every attribute and possible variants. Accordion in current docs present only 2 variants. Could we discuss it further? I will setup review call for us.

Reply:
It might be good to present all necessary variants for story for VR story reuse. I agree though, for some cases it can be too much. Ok, let's have a separate discussion for this.